src/Domain/ReadySession/Model/ReadySessionBase64Image.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace StepupReadId\Domain\ReadySession\Model;
+
+final class ReadySessionBase64Image
```

Comment:
Maybe move all `ReadySession` classes in a separate namespace and remove the ReadySession prefix from the classname. Not that important, but might make the code somewhat cleaner?

Reply:
That is the reason I add the ReadySession prefix. Sometimes I had same VO names with different behaviour and this let me to know who they belong (and let me avoid to use alias too).

Reply:
Yes, that's a clear benefit of this solution. I'm fine with this approach by the way.